pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+# Related Issue
+- Link for the issue
+- Description of the issue
```

Comment:
The description of the issue need not be required since it would be contained within the issue itself?